EitherTests/EitherTests.swift

```diff
@@ -51,3 +51,29 @@ final class EitherTests: XCTestCase {
 		XCTAssert(result)
 	}
 }
+
+final class EitherListTests: XCTestCase {
```

Comment:
I don’t know why it never occurred to me to use a new suite in the same file like this. This is a good trick for organization :+1: